### PR TITLE
Game version 3801

### PR DESCRIPTION
**Pull request by Garanas**

### CODE REVIEW

File: changelog/3801.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+- (#5865, #5929, #5934, #5938, #5936, #5937, #5935, #5928, #5930, 
+- #5942, #5966, #5968, #5969, #5965, #5967, #5964)
+    Update the categories of various units
+
+    These are used by key bindings and the UI to understand what the units are capable of.
+
+- (#5870, #5858, #5881) Improve the descriptions of a wide range of units
+
+- (#5873) Reduce the lifetime of scorchmarks created by forest fires
```
Comment (The-Balthazar) [inline changelog/3801.md:143]: This should mention the new times, xor not mention the old times.

Comment (Basilisk3) [reply changelog/3801.md:143]: Those were the ones which took 25 in-game hours to disappear.

File: changelog/3801.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+## Other Changes
+
+- (#5865, #5929, #5934, #5938, #5936, #5937, #5935, #5928, #5930, 
+- #5942, #5966, #5968, #5969, #5965, #5967, #5964)
+    Update the categories of various units
+
+    These are used by key bindings and the UI to understand what the units are capable of.
+
+- (#5870, #5858, #5881) Improve the descriptions of a wide range of units
+
```
Comment (Basilisk3) [inline changelog/3801.md:142]:
```suggestion

(#5939) Improve the naming scheme of all UEF drones and update their descriptions and Tech levels.

```

File: changelog/3801.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+## Other Changes
+
+- (#5865, #5929, #5934, #5938, #5936, #5937, #5935, #5928, #5930, 
+- #5942, #5966, #5968, #5969, #5965, #5967, #5964)
+    Update the categories of various units
+
+    These are used by key bindings and the UI to understand what the units are capable of.
+
+- (#5870, #5858, #5881) Improve the descriptions of a wide range of units
```
Comment (Basilisk3) [inline changelog/3801.md:141]:
```suggestion
- (#5870, #5858, #5881) Improve the descriptions of a wide range of units.
```

File: changelog/3801.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+- (#5878) Fix the Volcano being able to redirect strategic missiles.
+
+- (#5879) Fix various air units when landed being impossible to hit with non-splash weaponry.
+
+- (#5931) Fix the bug related to capture time and costs for factories that are building units.
+
+    The unit they were constructing would inflate the capture time and cost of the factory itself.
+
+- (#5958, #5979) Fix projectiles of the Fatboy and SACUs running out of lifetime before impact
```
Comment (Basilisk3) [inline changelog/3801.md:31]:
```suggestion
- (#5958, #5979) Fix projectiles of the Fatboy and SACUs with their respective range upgrades running out of lifetime before impact.
```

**Review by Basilisk3 — CHANGES_REQUESTED** [review]

File: changelog/3801.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+    The solution was to cache the templates so that they are only allocated once.
+
+- (#5852) The first decimal place of damage values are now visible in unit tooltips.
+
+- (#5849) Small tweaks to the Salvation to match the (visual) properties of other artillery.
+
+- (#5943) The Novax Satalite System is no longer disabled when air is restricted.
+
+    The Novac Satalite System can be restricted separately.
```
Comment (4z0t) [inline changelog/3801.md:165]: Typo `Novax`.

**Pull request merged**